library/src/main/java/com/mapzen/android/OverlayManager.java

```diff
@@ -334,6 +338,36 @@ public void clearRouteLine() {
     }
   }
 
+  /**
+   * Draws route line on the map for the points supplied.
+   */
+  public void drawTransitRouteLine(List<LngLat> points, String colorHex) {
+    if (transitRouteLineData == null) {
+      transitRouteLineData = mapController.addDataLayer(NAME_TRANSIT_ROUTE_LINE);
+    }
+    HashMap<String, String> properties = new HashMap<>();
+    properties.put(PROP_TYPE, PROP_LINE);
+
+    mapController.queueSceneUpdate("layers.mz_route_line_transit.draw.ux-transit-line-overlay.color",
```

Comment:
You should probably remove this line as my scene file code is already data driven, shouldn't need you mucking with updating the scene.